tests/phpunit/tests/block-bindings/register.php

```diff
@@ -13,7 +13,8 @@ class Tests_Block_Bindings_Register extends WP_UnitTestCase {
 
 	const TEST_SOURCE_NAME       = 'test/source';
 	const TEST_SOURCE_PROPERTIES = array(
-		'label' => 'Test source',
+		'label'              => 'Test source',
+		'get_value_callback' => 'test value',
```

Comment:
The callback should be a valid function name. That's another thing we should look at, adding a check whether the value passed is a [`callable`](https://www.php.net/manual/en/language.types.callable.php).

Reply:
Makes sense. I updated the tests in 9a1ed6e6d66f1b661acee13dd2f7c69ae03515a8

Reply:
I also added validation for whether `get_value_callback` is a callable.